README.md

```diff
@@ -256,41 +269,110 @@ scripts/build.sh
 
 `git clone https://github.com/fabric8io/fabric8-online.git`
 
-* Set some environment variables (or run `. ./scripts/setenv-for-deploy.sh`)
+* You may need to override the full path to the `fabric8-online` binary (provided by the `FABRIC8_ONLINE_PATH`
+environment variable), according to where you cloned the `fabric8-online` git repository :
+
```

Comment:
I would remove "may". And that is not a full path to a binary but a path to a repository where yaml for Che deployment are saved:
`s/You may need to override the full path to the fabric8-online binary/You need to set the local path to the fabric8-online git repo/`